src/Phpro/SoapClient/CodeGenerator/Util/TypeChecker.php

```diff
@@ -45,8 +45,9 @@ private static function isInternalPhpType(string $type): bool
      */
     private static function isClassType(string $type): bool
     {
-        //todo add check for available classes
-
+        if (preg_match('/^\\\[A-Z0-9]{1,}[a-zA-Z0-9\\\]+$/', $type)) {
```

Comment:
I've tested the regex above on https://regex101.com/ and it didn't seem to work well. The suggested one below works and covers more cases:

```suggestion
        if (preg_match('/^\\?[a-zA-Z0-9_]{1,}[a-zA-Z0-9_\\]+$/', $type)) {
```

Reply:
I've updated regular expression - lower case characters in the beginning of class and underscore are allowed. But I left check that class name has to be started with `\`